styleguide-app/Debug/Control/View.elm

```diff
@@ -62,6 +74,39 @@ viewExampleCode values =
             values
 
 
+generateEllieLink : { example | name : String, version : Int } -> String -> String
+generateEllieLink example code =
+    Url.Builder.crossOrigin "https://ellie-app.com/a/example/v1"
+        []
+        [ Url.Builder.string "title" example.name
+        , Url.Builder.string "elmcode" (generateElmExampleModule example code)
+        , -- TODO: a system will be required to keep these values in line with the elm json values
+          Url.Builder.string "packages" "elm/core@1.0.5"
+        , Url.Builder.string "packages" "elm/html@1.0.0"
+        , Url.Builder.string "packages" "rtfeldman/elm-css@17.0.5"
+        , Url.Builder.string "packages" "NoRedInk/noredink-ui@15.8.1"
```

Comment:
what if we wrote something like `NoRedInk/noredink-ui@<NRI_UI_VERSION>` or something here, then in the Netlify build script we just did some `sed` call? That'd address some of the brittleness you were talking about, yeah?

Reply:
or, even better, we could pull it from page flags!

Reply:
I was thinking maybe we could have a build step that took the noredink-ui elm.json and processed it into JSON!

(This is partly why I tagged Foxen as reviewers on this particular PR -- I thought you all were likely to have neat ideas for improvement here!)

Reply:
oooh, that's a good idea. Lemme make a sketch in a new PR.